docs/plugins/ingest-useragent.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[ingest-useragent]]
+=== Ingest Useragent Processor Plugin
+
+The Useragent processor extracts details from the useragent string a browser sends with its web requests. 
```

Comment:
I agree with the code naming and just pushed https://github.com/elastic/elasticsearch/pull/19074/commits/417feb7aae3e04af7c4843ed93deca026025b1be. I'd like to keep the plugin name as it is as that's what folks are used to from Logstash.